libs/test/SalesforceReactTest/src/com/salesforce/androidsdk/reactnative/util/ReactNativeTestHost.java

```diff
@@ -87,27 +83,12 @@ protected ReactInstanceManager createReactInstanceManager() {
                 // NB: Bundle is generated during build
                 .setBundleAssetName("index.android.bundle")
                 .setJSIModulesPackage(getJSIModulePackage())
-                .setUseDeveloperSupport(false)
-                .setNativeModuleCallExceptionHandler(new NativeModuleCallExceptionTestHandler());
+                .setUseDeveloperSupport(false);
 
         for (ReactPackage reactPackage : getPackages()) {
             builder.addPackage(reactPackage);
         }
 
         return builder.build();
     }
-
-
-    /**
-     * Implementation of NativeModuleCallExceptionHandler used for testing (when using bundle)
-     *
-     * Marks the current test running is marked as failed if any javascript error takes place
-     */
-    static class NativeModuleCallExceptionTestHandler implements NativeModuleCallExceptionHandler {
```

Comment:
That was only used when developing test to route the exception back to native land.
NativeModuleCallExceptionHandler no longer exists.